TombEngine/Scripting/Internal/ReservedScriptNames.h

```diff
@@ -139,6 +139,8 @@ static constexpr char ScriptReserved_GetInvItemCount[]				= "GetItemCount";
 static constexpr char ScriptReserved_SetInvItemCount[]				= "SetItemCount";
 static constexpr char ScriptReserved_GetMoveableByName[]			= "GetMoveableByName";
 static constexpr char ScriptReserved_GetStaticByName[]				= "GetStaticByName";
+static constexpr char ScriptReserved_GetMoveablesBySlot[]			= "GetMoveablesBySlot";
+static constexpr char ScriptReserved_GetStaticsById[]				= "GetStaticsById";
```

Comment:
I would rename it to `GetStaticsBySlot` as well to maintain consistency.

Reply:
done